CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ All notable changes to this project will be documented in this file. This projec
 - New option `--no-files` added to `multidev:create` which will skip the duplication of files from the source environment. (#2050)
 - New option `no-db` added to `Environments::create` which will skip the duplication of the database from the source environment. (#2050)
 - New option `no-files` added to `Environments::create` which will skip the duplication of files from the source environment. (#2050)
+- New command `self:plugin:install` to install Terminus plugins. (#2054)
+- New command `self:plugin:list` to list installed Terminus plugins. (#2054)
+- New command `self:plugin:search` to locate Terminus plugins to install. (#2054)
+- New command `self:plugin:uninstall` to uninstall Terminus plugins. (#2054)
+- New command `self:plugin:update` to update already-installed Terminus plugins. (#2054)
```

Comment:
Will `plugin:i` work or is `self:` necessary?  Less is more, right?

Reply:
My concern with naming them so is that a user may think that Terminus' plugin commands manage WP(-CLI) plugins. There's always `s:p:li`

Reply:
Maybe it's just me but I'm not sure how `self` would help someone who is confused with WordPress or Terminus plugins.  Could we have a big notice in the usage message something along the lines of `Do not confuse the plugin command with WordPress plugins.  This command refers to Terminus plugins.`?